packages/integration-tests/src/config.ts

```diff
@@ -38,4 +38,8 @@ export class Config {
   public static StateCommitmentChainContractAddress(): string {
     return process.env.STATE_COMMITMENT_CHAIN_CONTRACT_ADDRESS
   }
+
+  public static l1ContractDeploymentPrivateKey(): string {
+    return process.env.L1_CONTRACT_DEPLOYMENT_PRIVATE_KEY
+  }
 }
```

Comment:
This file is duplicated across the packages right now, but really I think we want to import the `Config` from `@eth-optimism/contracts`, need to export it because it was undefined when I tried importing it

Reply:
See https://github.com/ethereum-optimism/optimism-monorepo/issues/313